compute/wps/webapp/src/app/configure/process.ts

```diff
@@ -69,11 +73,15 @@ export class Process {
       result: this.newUID(),
     };
 
-    operation['input'] = this.inputs.map((item: File) => {
-      return item.uid;
-    });
+    if (this.inputs != null) {
```

Comment:
should this throw an error on the else case?

Reply:
This will be completely changed in the workflow branch so I wouldn't worry about it.